### DISCUSSION TITLE
Cannot import 'main' from xpore-dataprep

### DISCUSSION THREAD

**Discussion by esh97** [GoekeLab/xpore#145] [Q&A] [answered]
I am trying to run this program on some sample data but when I try to run the command xpore-dataprep I get this error message returned: File "/home/anaconda3/bin/xpore-dataprep", line 6, in <module>
    from xpore.scripts.dataprep import main
ImportError: cannot import name 'main' from 'xpore.scripts.dataprep'

**Comment by yuukiiwa** [comment, marked as answer]
Hi @esh97, 

Thr xpore-dataprep command is from xpore version 1.0. We would suggest to update your xpore installation to the latest version 2.1.

Thanks!

Best wishes,
Yuk Kei

**Reply by esh97** [reply]
Thank you! I have the 2.1 version but is there a guide to the commands in this version? Is there still a data-prep command?
